ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/sflow.schema.yml

```diff
@@ -51,14 +59,33 @@ keys:
           keys:
             destination:
               type: str
-              description: Sflow Destination IP
+              description: Sflow Destination IP Address
             port:
               type: int
               description: Port Number
               convert_types:
               - str
+      source:
+        type: str
+        description: |
+          "Source IP Address
+          source and source_interface are mutually exclusive. If both are defined, source_interface takes precedence"
```

Comment:
```suggestion
        description: |
          Source IP Address.
          "source" and "source_interface" are mutually exclusive. If both are defined, "source_interface" takes precedence.
```

Reply:
comments > code additions.. I really need to pay more attention. it's been a while since the last commit and feels like starting from.. not actually zero, but kinda close.